disko-install

```diff
@@ -253,7 +253,8 @@ main() {
   # nix copy uses up a lot of memory and we work around issues with incorrect checksums in our store
   # that can be caused by using closureInfo in combination with multiple builders and non-deterministic builds.
   # Therefore if we have a blank store, we copy the store paths and registration from the closureInfo.
-  if [[ ! -d "${mountPoint}/nix/store" ]]; then
+  if [[ ! -f "${mountPoint}/nix/var/nix/db/db.sqlite" ]]; then
+    /nix/var/nix/db/db.sqlite
```

Comment:
Is this valid semantically? It looks like you're executing the db, but surely that is not possible, right?

Reply:
Yeah. I don't know how this got there.